### DISCUSSION TITLE
Saving the screen layout

### DISCUSSION THREAD

**Discussion by ChuckBland** [jasonacox/Powerwall-Dashboard#497] [General]
Is it possible to save the screen layout of the dashboard so that it will restart in the configuration in which it was stopped?

It's a pain to have to re-arrange the screen with each start.

Chuck

**Comment by jasonacox** [comment]
Yes, that is built into Grafana. You need to use the save button - top right:

<img width="170" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/d21a2ac3-44ea-4dd0-ae13-6955a8f8d362">

**Comment by ChuckBland** [comment]
Thanks for the inspiration.

When I went looking for that icon and didn't find it, I noticed the word
"kiosk" in the browser URL and thought "I wonder what happens if I press
ESC..... oh, look, there's the save icon....."

Problem solved.

Chuck
